src/components/pages/resources/hero/hero.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import Link from 'components/shared/link/link';
+
+import cover3 from './images/cocver-ebpf-2.png';
```

Comment:
Looks like a typo `cocver-ebpf-2`, and I would recommend to use Static image instead of img